src/elements/Button/Button.module.css

```diff
@@ -29,6 +29,40 @@
     font-size: var(--font-size-lg);
   }
 
+  &.filledDates {
+    color: var(--button-background);
```

Comment:
These should be generic - also not coupled to the date. You can provide CSS there.